.github/workflows/build.yml

```diff
@@ -75,7 +75,7 @@ jobs:
   instrumentation-tests:
 
     name: Instrumentation tests
-    runs-on: macos-12
+    runs-on: ubuntu-22.04
```

Comment:
I remember having a conversation with Mariano about why we're using MacOS to run our android e2e tests. Back then, the android emulator had [terrible support for linux](https://github.com/ReactiveCircus/android-emulator-runner/issues/81#issuecomment-658291981)
Is it good enough to switch to linux now?

Reply:
Ah, thanks for the context @m5r!  I had not been able to understand why this was originally on macOS.  The good news is that since the GH Linux runners [now support KVM hardware acceleration](https://github.blog/changelog/2023-02-23-hardware-accelerated-android-virtualization-on-actions-windows-and-linux-larger-hosted-runners/), the Linux runners are actually [much faster for Android builds](https://github.com/ReactiveCircus/android-emulator-runner/issues/46#issuecomment-1899848624) than macOS (this improvement was noticeable even for our tests).

Reply:
Also worth noting that the [example usage documentation](https://github.com/ReactiveCircus/android-emulator-runner?tab=readme-ov-file#usage--examples) for `ReactiveCircus/android-emulator-runner` now suggest using Linux.